index.html

```diff
@@ -5,9 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 <body>
-    <header></header>
-    <section></section>
+    <header>
+        <div class="status-bar"></div>
+        <div class="toolbar">
+            <div class="logo-placeholder">
+                <p class="name white-color">Thomas Obuya</p>
+            </div>
+            <div class="menu-placeholder">
+                <button class="menu">
+                    <img src="menu.svg" alt="Menu">
+                </button>
+            </div>
```

Comment:
- [ ] I think it will be better if you make your website semantic by wrapping your hamburger in a `<nav>` because your hamburger will contain your navigation items thereby enabling screen readers and search engines to understand that the section is meant for navigations which will increase your website accessibility 👍🏽